static/app/views/profiling/landing/profilesChartWidget.tsx

```diff
@@ -116,7 +115,6 @@ export function ProfilesChartWidget({
     <WidgetContainer height={widgetHeight}>
       <HeaderContainer>
         {header ?? <HeaderTitleLegend>{t('Profiles by Percentiles')}</HeaderTitleLegend>}
-        <Subtitle>{t('P50(), P75(), P95(), P99() over time')}</Subtitle>
```

Comment:
This is just duplicate content as the chart legend so 🔪 it

Reply:
I recall we added this because the title alone wasn't clear enough not sure how folks feel about removing it now just because the legend has the same info

Reply:
@Zylphrex let me change the title to `Profiles by percentiles over time`, but I don't think we need to duplicate the pxx info.